cmd/cli/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"encoding/json"
+
+	"github.com/segmentio/analytics-go"
+	"github.com/segmentio/conf"
+)
+
+func main() {
+	var config struct {
+		WriteKey   string `conf:"writeKey"   help:"The Segment Write Key of the project to send data to"`
+		Type       string `conf:"type"       help:"The type of the message to send"`
+		UserID     string `conf:"userId"     help:"Unique identifier for the user"`
+		GroupID    string `conf:"groupId"    help:"Unique identifier for the group"`
+		Traits     string `conf:"traits"     help:"Metadata associated with the user"`
+		Event      string `conf:"event"      help:"Name of the track event"`
+		Properties string `conf:"properties" help:"Metadata associated with an event, page or screen call"`
+		Name       string `conf:"name"       help:"Name of the page/screen"`
+	}
+	conf.Load(&config)
+
+	callback := newCallback()
+
+	client, err := analytics.NewWithConfig(config.WriteKey, analytics.Config{
+		BatchSize: 1,
+		Callback:  callback,
+	})
+	if err != nil {
+		panic(err)
```

Comment:
Is it worth panicking here? it'll give a stack trace that isn't very interesting to the end user.

How about one of these options?
```go
fmt.Println(err)
os.Exit(1)
```
```go
log.Fatal(err)
```

Reply:
Didn't pay too much attention to error messages since this is just for testing atm, but will update!

Reply:
Done!